app/core/social-handlers/GPlusHandler.coffee

```diff
@@ -20,13 +20,27 @@ scope = 'https://www.googleapis.com/auth/plus.login email'
 module.exports = GPlusHandler = class GPlusHandler extends CocoClass
   constructor: ->
     @accessToken = storage.load GPLUS_TOKEN_KEY, false
+    window.onGPlusLogin = _.bind(@onGPlusLogin, @)
     super()
 
-  subscriptions:
-    'auth:logged-in-with-gplus':'onGPlusLogin'
-    'auth:gplus-api-loaded':'onGPlusLoaded'
+  token: -> @accessToken?.access_token
 
-  onGPlusLoaded: ->
+  loadAPI: ->
+    return if @loadedAPI
+    @loadedAPI = true
+    (=>
+      po = document.createElement('script')
+      po.type = 'text/javascript'
+      po.async = true
+      po.src = 'https://apis.google.com/js/client:platform.js?onload=onGPlusLoaded'
+      s = document.getElementsByTagName('script')[0]
+      s.parentNode.insertBefore po, s
+      window.onGPlusLoaded = _.bind(@onLoadAPI, @)
```

Comment:
I don't believe so, since `async = true`. I like to keep these in chronological order, and to encourage building and using systems such that you don't have to worry about it.